src/components/FilterAnnotModal/FilterAnnotModal.js

```diff
@@ -319,10 +320,10 @@ const FilterAnnotModal = ({ coAssessors, notesShareTypesMap, shareTypeColors })
                 }
                 id={val}
                 onChange={e => {
-                  if (statusFilter.indexOf(e.target.getAttribute('id')) === -1) {
-                    setStatusFilter([...statusFilter, e.target.getAttribute('id')]);
+                  if (shareTypeFilter.indexOf(e.target.getAttribute('id')) === -1) {
+                    setShareTypeFilter([...shareTypeFilter, e.target.getAttribute('id')]);
                   } else {
-                    setStatusFilter(statusFilter.filter(status => status !== e.target.getAttribute('id')));
+                    setShareTypeFilter(shareTypeFilter.filter(status => status !== e.target.getAttribute('id')));
                   }
```

Comment:
Should this be moved to a handler? It would cleanup the code a bit :)

Reply:
To keep changes as low as possible, I'll leave this as it is.